src/pages/settings/Profile/ProfilePage.js

```diff
@@ -199,10 +205,16 @@ class ProfilePage extends Component {
                     onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
                     onCloseButtonPress={() => Navigation.dismissModal(true)}
                 />
-                <ScrollView style={styles.flex1} contentContainerStyle={styles.p5}>
+                <Form
+                    style={[styles.flexGrow1, styles.ph5]}
+                    formID={CONST.PROFILE_SETTINGS_FORM}
```

Comment:
FYI now'days we put the form IDs here: https://github.com/Expensify/App/blob/ff5affde1eec531768e86744fd5f084aa66ec7cf/src/ONYXKEYS.js#L171-L176